common.js

```diff
@@ -6,6 +6,8 @@ const stream = require('stream')
 const crypto = require('crypto')
 const core = require('@actions/core')
 const tc = require('@actions/tool-cache')
+const exec = require('@actions/exec')
+const common = require('./common')
```

Comment:
This is a require of this file, it seems safer to avoid it, as I guess it might cause some weird stuff.
I'll fix it.